CMakeLists.txt

```diff
@@ -52,12 +71,21 @@ add_library(timers
   perf_mod.F90
   perf_utils.F90)
 
-target_compile_definitions(timers PRIVATE FORTRANUNDERSCORE HAVE_MPI LINUX PERFSTUBS_USE_TIMERS)
-target_link_libraries(timers PUBLIC MPI::MPI_C global_settings)
+if (MPI_FOUND)
+    target_compile_definitions(timers PRIVATE HAVE_MPI )
+endif (MPI_FOUND)
+
+target_compile_definitions(timers PRIVATE FORTRANUNDERSCORE LINUX PERFSTUBS_USE_TIMERS)
+
+if (MPI_FOUND)
+    target_link_libraries(timers PUBLIC MPI::MPI_C global_settings)
+endif (MPI_FOUND)
+
```

Comment:
Merging the two `if (MPI_FOUND)` blocks into one would make it a bit more readable.

Unrelated, but as I see the `global_settings` thing here, I think I need to double check whether that's just a remnant that should go away.

Reply:
> Merging the two `if (MPI_FOUND)` blocks into one would make it a bit more readable.
> 
> Unrelated, but as I see the `global_settings` thing here, I think I need to double check whether that's just a remnant that should go away.

Good point, I'll fix that.